src/Model/Comment.php

```diff
@@ -27,14 +27,16 @@ class Comment extends Entity
     /**
      * Comment constructor.
      *
-     * @param array $data
+     * @param null|array $data
      */
-    public function __construct(array $data)
+    public function __construct(?array $data)
     {
-        $this->hydrate($data);
-        if (!isset($data['created_at'], $data['updated_at'])) {
-            $this->setCreatedAt();
-            $this->setUpdatedAt();
+        if (null !== $data) {
```

Comment:
No needed. You must set createdAt and updatedAt directly on trait. Hydrate must be call on parent entity